packages/mds-policy-author/api.ts

```diff
@@ -23,7 +23,8 @@ import {
   UUID_REGEX,
   NotFoundError,
   BadParamsError,
-  AlreadyPublishedError
+  AlreadyPublishedError,
+  DependencyMissingError
```

Comment:
These two errors are used in a complimentary fashion, but their names don't really convey that. Maybe they could be `DependencyConflictError` (409) and `DependencyMissingError` (404)? Or `AlreadyPublishedError/NotPublishedError`?